apps/app_rpt/rpt_functions.c

```diff
@@ -1714,28 +1714,25 @@ int function_cop(struct rpt *myrpt, char *param, char *digitbuf, int command_sou
 			rpt_telemetry(myrpt, COMPLETE, NULL);
 		}
 		return DC_COMPLETE;
-	case 63:					/* send pre-configured APRSTT notification */
-	case 64:
-		if (argc < 2)
-			break;
-		if (!myrpt->p.aprstt)
-			break;
-		if (!myrpt->p.aprstt[0])
-			ast_copy_string(fname, APRSTT_PIPE, sizeof(fname) - 1);
-		else
-			snprintf(fname, sizeof(fname) - 1, APRSTT_SUB_PIPE, myrpt->p.aprstt);
-		fp = fopen(fname, "w");
-		if (!fp) {
-			ast_log(LOG_WARNING, "Can not open APRSTT pipe %s\n", fname);
+	case 63:					/* send pre-configured APRStt notification */
+	case 64:					/* same, but quiet */
+		if (argc < 2) {
 			break;
 		}
-		if (argc > 2)
-			fprintf(fp, "%s %c\n", argv[1], *argv[2]);
-		else
-			fprintf(fp, "%s\n", argv[1]);
-		fclose(fp);
-		if (myatoi(argv[0]) == 63)
-			rpt_telemetry(myrpt, ARB_ALPHA, (void *) argv[1]);
+		
+		if (!ast_custom_function_find("APRS_SENDTT")) {
+			ast_log(LOG_WARNING, "app_gps is not loaded.  APRSTT failed\n");
+			return DC_COMPLETE;
+		}
+		memset(func, 0, sizeof(func));
+		snprintf(func, sizeof(func) -1, "APRS_SENDTT(%s,%c)", !myrpt->p.aprstt ? "general" : myrpt->p.aprstt, 
+			argc > 2 ? argv[3][0] : ' ');
```

Comment:
Looking at this again, Is something wrong with this test?  argv[3] is argument 4 right?  should it be `> 3?`